Cake.Recipe/Content/addins.cake

```diff
@@ -2,25 +2,25 @@
 // ADDINS
 ///////////////////////////////////////////////////////////////////////////////
 
-#addin nuget:?package=Cake.Codecov&version=0.5.0
-#addin nuget:?package=Cake.Coveralls&version=0.9.0
-#addin nuget:?package=Cake.Figlet&version=1.2.0
-#addin nuget:?package=Cake.Git&version=0.19.0
-#addin nuget:?package=Cake.Gitter&version=0.10.0
-#addin nuget:?package=Cake.Graph&version=0.6.0
-#addin nuget:?package=Cake.Incubator&version=3.1.0
-#addin nuget:?package=Cake.Kudu&version=0.8.0
-#addin nuget:?package=Cake.MicrosoftTeams&version=0.8.0
-#addin nuget:?package=Cake.ReSharperReports&version=0.10.0
+#addin nuget:?package=Cake.Codecov&version=0.6.0
```

Comment:
Suggest to upgrade to version `0.7.0` as this version provides support for the latest `Codecov` nuget package.

```suggestion
#addin nuget:?package=Cake.Codecov&version=0.7.0
```

Reply:
0.6.0 was the latest version at the time this PR was created and the add-in Discoverer will not make any new recommandation until this PR is merged. Therefore, this PR should be merged and within a few hours a new PR will be automatically created to upgrade the reference to the latest.

Reply:
@AdmiringWorm  Somehow I missed the fact that you proposed a code change. Your change could simply be accepted in this PR which would avoid having to wait a few hours for a subsequent PR.